metaflow/tutorials/02-statistics/README.md

```diff
@@ -6,13 +6,16 @@ later examples to improve our playlist generator. You can optionally use the
 Metaflow client to eyeball the results in a Notebook, and make some simple
 plots using the Matplotlib library.**
 
+Please note that Episode 04, a follow-on to this episode, requires Pandas version 0.24.2.
+If you have a more recent version of Pandas and cannot upgrade, you will not be able to run
```

Comment:
Can we change the wording to `Please make sure that you install or upgrade/downgrade to pandas 0.24.2`